src/storage/redis_db.cc

```diff
@@ -721,6 +721,51 @@ rocksdb::Status Database::Rename(const std::string &key, const std::string &new_
 
   if (key == new_key) return rocksdb::Status::OK();
 
+  return moveInternal(ns_key, new_ns_key, type);
+}
+
+rocksdb::Status Database::Move(const std::string &key, const std::string &new_ns, bool *ret) {
```

Comment:
Seems the only difference between the RENAME and MOVE is how to construct its new key. The rename command requires the same namespace and a different key, and the move command requires a different namespace but the same key. So I think we need to move the construction of the new key out of Database::Rename, and then both of rename/move can share the same implementation.